depends/packages/libpng.mk

```diff
@@ -0,0 +1,23 @@
+package=libpng
+$(package)_version=1.6.34
+$(package)_download_path=http://ftp.osuosl.org/pub/libpng/src/libpng16/
+$(package)_file_name=$(package)-$($(package)_version).tar.gz
+$(package)_sha256_hash=574623a4901a9969080ab4a2df9437026c8a87150dfd5c235e28c94b212964a7
+$(package)_dependencies=zlib
+
+define $(package)_set_vars
+  $(package)_config_opts=--enable-static --disable-shared
```

Comment:
You can pass `--disable-dependency-tracking` here (as we do to all other packages that support it). Had a quick look at the configure options and it doesn't seem like there's anything obvious we should be disabling, like building documentation.